app/views/claims/_school_search.html.erb

```diff
@@ -37,7 +37,7 @@
           </span>
         </p>
 
-        <% if school_id_param == :claim_school_id %>
+        <% if school_id_param == :claim_school_id && params[:additional_school].blank? %>
```

Comment:
The use case for `blank?` is when the variable might be a blank string. In this case, it's either a parameter that is present and `true` or it's not, so `params[:additional_school]` will suffice.